src/js/components/Accordion/__tests__/__snapshots__/Accordion-test.tsx.snap

```diff
@@ -252,7 +252,7 @@ exports[`Accordion AccordionPanel 1`] = `
           </div>
         </button>
         <div
-          aria-labelledby="«r1»"
+          aria-labelledby="_r_1_"
```

Comment:
This comes from React 

“Use underscore instead of : for IDs generated by useId”

https://github.com/facebook/react/releases/tag/v19.2.0
can be seen under notable changes